### PR TITLE
adds Greek salad to lunch

### PR DESCRIPTION
**Pull request by cesve**
### This PR adds | fixes:
- Adds Greek Salad to Meatballs repo

**How to prepare for test**:
- [ ] `ssh` to ...
- [ ] Install on stage:
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
-

### Expected outcome:
- [ ]

### Review:
- [ ] Code approved by
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: docs/lunch/Salad.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+![salad](../img/salad.jpeg)
+
+## Ingredients
+- Tomato
+- Cucumber
+- Red onion
+- Feta cheese
+- Olives
+- Olive oil
```
Comment (idalindegaard) [inline docs/lunch/Salad.md:11]: Maybe add some herbs?

**Review by idalindegaard — APPROVED** [review]
Yum!

**Comment by cesve** [comment]
Take a look at this @idalindegaard

File: docs/lunch/Salad.md (hunk 1 above)
Comment (VasKyriak) [inline docs/lunch/Salad.md:11]: Extra virgin Olive Oil
```suggestion
- Extra virgin Greek Olive Oil
```

**Pull request merged**